store/file/q4.go

```diff
@@ -0,0 +1,183 @@
+package file
+
+import (
+	"bufio"
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/celestiaorg/rsmt2d"
+
+	"github.com/celestiaorg/celestia-node/share"
+	eds "github.com/celestiaorg/celestia-node/share/new_eds"
+	"github.com/celestiaorg/celestia-node/share/shwap"
+)
+
+var _ eds.AccessorStreamer = (*Q4)(nil)
```

Comment:
Do we really need it? Seems like Q4 implements only AxisHalf and Close. And is not going to be used as generic Accessor anywhere. Does not seem that we need Q4 to be Accessor and can remove non-implemented methods.

Reply:
I use it as accessor type in ODSQ4